otherlibs/configurator/test/blackbox-tests/pkg-config-args.t/run.t

```diff
@@ -0,0 +1,34 @@
+These tests show that setting `PKG_CONFIG_ARGN` passes extra args to `pkg-config`
+
+  $ dune build 2>&1 | awk '/run:.*bin\/pkg-config/{a=1}/stderr/{a=0}a'
+  run: $TESTCASE_ROOT/_build/install/default/bin/pkg-config --print-errors gtk+-quartz-3.0
+  -> process exited with code 0
+  -> stdout:
+   | gtk+-quartz-3.0
```

Comment:
Not sure if this is reproducible. Personally I would just write a dummy `pkg-config` shell script that prints its args and put it in PATH for this test.

Reply:
I copied it from `pkg-config-quoting.t`, which is. Note the redefined `pkg_config.exe` in the test that makes it reproducible.

Reply:
This is reproducible because of the overridden `pkg_config.ml` included in the test. I made it query / output "dummy-pkg" to make it more explicit that it's not actually calling a real `pkg-config`.